stage6/02-adi-update-tools/00-run.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+
+on_chroot << EOF
+git clone https://github.com/analogdevicesinc/linux_image_ADI-scripts
+
+pushd linux_image_ADI-scripts
+chmod +x adi_update_tools.sh
+./adi_update_tools.sh
```

Comment:
is there a way where we can parametrize the `./adi_update_tools.sh` script?
https://github.com/analogdevicesinc/linux_image_ADI-scripts/blob/master/adi_update_tools.sh#L191
it looks like, it will pick by default the latest release branch; which may be fine

but we may also want to have a better control over this [at some point] to allow to build images with development stuff;

Reply:
i could use the 2019_R1 as a parameter. would that be more suited ?

Reply:
if we can set it as a global parameter it somwhere, it would be great